swagger.yaml

```diff
@@ -366,17 +410,20 @@ paths:
         schema:
           type: string
       responses:
+        400:
+          description: 'Error in request.  A list of errors will be returned'
         404:
           description: 'Device does not exist'
         200:
-          description: 'successful operation'
+          description: 'Successful operation'
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Device'
-  /api/v1/devices/disconnect/{guid}:
+        500:
+          description: 'Internal server error'
     delete:
-      description: Forces a specified device to disconnect from the MPS. Device will reconnect again if it is able.
+      description: Removes the device with a specific GUID and all device information from the database 
```

Comment:
Might want to add that this will also prevent the device from connecting even if a device has not yet been unprovisioned.

Reply:
yup, that is new since these changes.  Good catch